src/View.Uno.Shared/Controls/AnimatedHeaderControl/AnimatedHeaderControl.cs

```diff
@@ -190,27 +190,25 @@ private void SubscribeToViewChanged()
 
 				if (_scrollViewer != null)
 				{
-					void onViewChanged(object sender, ScrollViewerViewChangedEventArgs e)
-					{
-						AbsoluteOffset = _scrollViewer.VerticalOffset;
-					}
-
 					_scrollViewer.ViewChanged += onViewChanged;
 #if __ANDROID__ || __IOS__
-					RegisterLoadActions(loaded: () =>
```

Comment:
Why change this logic? Seems like unnecessary risks.

Reply:
It's necessary because it was removed from Uno 5
https://unoplatformdocstaging.z13.web.core.windows.net/articles/migrating-to-uno-5.html#registerloadactions-has-been-removed

Reply:
oh ok